src/CPULSSolver.cpp

```diff
@@ -197,7 +197,7 @@ void CPULSSolver::computeFSRSources(int iteration) {
   CPUSolver::computeFSRSources(iteration);
 
   int num_coeffs = 3;
-  if (_solve_3D)
+  if (_SOLVE_3D)
```

Comment:
If THREED macro is used, will this branch be deleted by optimization?

Reply:
I think so. Using this macro optimization gets a small speedup, which I think is due to that

Reply:
I suppose it should. Maybe we can check the optimization report to make sure.

Reply:
I just checked. It doesn't say anything about the branch, but I'm confident it does delete the else branch here, because :
- without the THREED flag, the optimization report reports the optimization of the 2D for loop
- with the flag, it doesn't mention these loops anywhere, so they got ignored at compile time